integration-tests/operator/cmd/validate_instrumentation_vars.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func main() {
+
+	args := os.Args
+	namespace := args[1]
+	jsonPath := args[2]
+	success := verifyInstrumentationEnvVariables(namespace, jsonPath)
+	if !success {
+		fmt.Println("Instrumentation Annotation Injection Test: FAIL")
+		os.Exit(1)
+	} else {
+		fmt.Println("Instrumentation Annotation Injection Test: PASS")
+	}
+}
+
+func verifyInstrumentationEnvVariables(namespace string, jsonpath string) bool {
+
+	var args []string
+	args = []string{"get", "pods", "-n", namespace, "-l", "app=nginx", "-o=jsonpath='{.items[*].metadata.name}'"}
```

Comment:
Using the k8s go client would have been a lot cleaner. You would need to do all the unmarshalling etc. and now we have a weird mix of shell and go.

Reply:
I agree. We can make more similar to other integ test by using k8s client